osi_trafficlight.proto

```diff
@@ -78,7 +78,11 @@ message TrafficLight
     {
         // The semantic color of the traffic light.
         //
-        // \note The color types represent the semantic color classification of a 
+        // \attention DEPRECATED: This color enum will be removed in version
+        // 4.0.0. Use the field \c #phase (\c Phase) of
+        // \c Classification instead.
```

Comment:
I wonder if this really should be deprecated for OSI. For OpenSCENARIO and the agent model use case it is probably a good idea to describe traffic lights with a phase as you suggest in this PR. But the (semantic) color might still be an interesting input for sensor models... Maybe it could be similar to the separation of logical and physical information for lanes which was introduced in [PR599](https://github.com/OpenSimulationInterface/open-simulation-interface/pull/599).